src/config/index.js

```diff
@@ -7,6 +7,7 @@ const initialState = {
 const SESSION_PREFIX = 'pkce-verifier';
 
 const KINDE_SITE_URL = process.env.KINDE_SITE_URL;
+const KINDE_API_PATH = process.env.KINDE_API_PATH || '/api/auth';
```

Comment:
Hey @kengreeff, thanks for the PR! Would you mind naming this variable `KINDE_API_PATH` to `KINDE_AUTH_API_PATH` as we'll soon be adding other non-auth features which will have different paths

Reply:
Done :)